packages/ui/src/components/LogLine/index.tsx

```diff
@@ -35,10 +36,10 @@ export const LogLine = ({ date, level = 'info', message }: LogLineProps) => {
   }, [level]);
 
   return (
-    <div className={`flex justify-between w-full px-2 py-1 rounded-md group ${containerStyle}`}>
+    <div className={cx([`flex justify-between w-full px-2 py-1 rounded-md group`, containerStyle])}>
```

Comment:
We can use single quotes here instead of template literals. Could you make this change in others files too?

```suggestion
    <div className={cx(['flex justify-between w-full px-2 py-1 rounded-md group', containerStyle])}>
```